main.js

```diff
@@ -176,38 +177,119 @@ const articleCardBuilder = (arr) => {
 //Build all Product cards
 const buildProductCard = (arr) => {
   domString = '';
-
   for (let i = 0; i < arr.length; i++) {
-      domString += `
-    <div class="col-6 col-md-3">
-    <div class="card testcard">
-      <div class="card-body p-2">
-        <img class="card-img-top adj" src="${arr[i].imgUrl[0].url}" alt="">
-        <h5 class="card-title">${arr[i].name}</h5>
-        <p class="card-text testCardText">$${arr[i].price}</p>        
-        <p class="card-text">${arr[i].model}</p>
-        <a id ="${arr[i].id}" onClick= "detailsCard(this.id)" href="#" class="btn btn-outline-warning btnDetails">Details</a>        
-      </div>
-    </div>
-    </div>
-    `;
+    domString += `
+       <div class="col-6 col-md-3">
```

Comment:
I would add another class to this for `mb-3` to add margin to the bottom of each of the cards.  Otherwise, the cards on the lower rows are smashed against those on the top row.

![Screen Shot 2020-05-16 at 9 42 38 AM](https://user-images.githubusercontent.com/10491407/82122597-37f6e680-975a-11ea-8f86-11229e2ca289.png)